README.md

```diff
@@ -71,6 +71,20 @@ ClusterServiceVersions can be collected into `CatalogSource`s which will allow a
 
 Learn more about the components used by OLM by reading about the [architecture] and [philosophy].
 
+# Overall OLM Chart
```

Comment:
I think "Component Diagram" or "Architecture Diagram" would be more clear.

Reply:
Done